tests/CustomJointMultiBodyTests.cc

```diff
@@ -528,14 +527,13 @@ TEST_FIXTURE ( CustomJointMultiBodyFixture, UpdateKinematics ) {
                       qddot.at(idx));
 
 
-    Matrix3d Eref = reference_model.at(idx).X_base[
-                      reference_body_id.at(idx).at(NUMBER_OF_BODIES-1)
-                      ].E;
-    Matrix3d Ecus = custom_model.at(idx).X_base[
-                      custom_body_id.at(idx).at(NUMBER_OF_BODIES-1)
-                      ].E;
-
-    Matrix3d Eerr = Eref-Ecus;
+    // Matrix3d Eref = reference_model.at(idx).X_base[
+    //                   reference_body_id.at(idx).at(NUMBER_OF_BODIES-1)
+    //                   ].E;
+    // Matrix3d Ecus = custom_model.at(idx).X_base[
+    //                   custom_body_id.at(idx).at(NUMBER_OF_BODIES-1)
+    //                   ].E;
+    // Matrix3d Eerr = Eref-Ecus;
```

Comment:
If these are not used, why keep them commented out?

Reply:
Hi Martin,
as we are about to redo the unit test, I left the things commented for the next to pick it up.
Best
Manuel